app/src/main/java/edu/uc/mille5bm/smartnutrition/MainActivity.kt

```diff
@@ -9,8 +9,8 @@ class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
-        findViewById<View>(R.id.button1).setOnClickListener { startActivity(Intent(applicationContext, NutritionInfoActivity::class.java)) }
-        findViewById<View>(R.id.button2).setOnClickListener { startActivity(Intent(applicationContext, CalorieCounterActivity::class.java)) }
-        findViewById<View>(R.id.button3).setOnClickListener { startActivity(Intent(applicationContext, HealthTrackerActivity::class.java)) }
+        findViewById<View>(R.id.btn_nutrition).setOnClickListener { startActivity(Intent(applicationContext, NutritionInfoActivity::class.java)) }
+        findViewById<View>(R.id.btn_calorie).setOnClickListener { startActivity(Intent(applicationContext, CalorieCounterActivity::class.java)) }
+        findViewById<View>(R.id.btn_health).setOnClickListener { startActivity(Intent(applicationContext, HealthTrackerActivity::class.java)) }
```

Comment:
Renamed buttons to be more descriptive

Reply:
Definitely... readable is good.  You'll forget what button1, button2, and button3 are for.

I'd go one step further and  use lowerCamelCase.